client/components/tooltip/docs/example.jsx

```diff
@@ -2,47 +2,47 @@
 /**
  * External dependencies
  */
-import React from 'react';
-import PureRenderMixin from 'react-pure-render/mixin';
+import React, { PureComponent } from 'react';
 
 /**
  * Internal dependencies
  */
 import Tooltip from 'components/tooltip';
 
-const TooltipInstance = React.createClass( {
-	mixins: [ PureRenderMixin ],
+class TooltipInstance extends PureComponent {
+	static displayName = 'Tooltip';
 
-	componentWillMount() {
+	constructor() {
+		super();
```

Comment:
I'm not sure it's explicitly required, but probably a good idea to pass the constructor arguments (i.e. props) to the super.

```js
constructor( props ) {
	super( props );
```

Or:

```js
constructor() {
	super( ...arguments );
```